prime-dotnet-webapi/Models/Feedback.cs

```diff
@@ -13,7 +13,8 @@ public class Feedback : BaseAuditable
         public int EnrolleeId { get; set; }
 
         [Required]
-        public string route { get; set; }
+        [Column("route")]
+        public string Route { get; set; }
```

Comment:
Decided to just keep the DB column lower case to avoid making another migration. We can update this when/if Feedback is looked at again.